src/test/java/io/kestra/plugin/ai/rag/ChatCompletionTest.java

```diff
@@ -163,4 +165,40 @@ void rag_givenTavilyWebSearch() throws Exception {
         var ragOutput = rag.run(runContext);
         assertThat(ragOutput.getCompletion()).isNotNull();
     }
+
+    @EnabledIfEnvironmentVariable(named = "RAPID_API_KEY", matches = ".*")
+    @Test
+    void tools_givenJudge0JavaScript() throws Exception {
+        RunContext runContext = runContextFactory.of(Map.of(
+            "modelName", "tinydolphin",
+            "endpoint", ollamaEndpoint,
+            "apikey", "1e3b194adamshae3a588f676c08ep1d9f6ejsna0ce4f7eb0ea", // RAPID_API_KEY,
```

Comment:
If it's yours, better to hide that, and destroy your keys as you just leaked it!